cli/azd/test/functional/cli_test.go

```diff
@@ -310,7 +310,7 @@ func Test_CLI_ProjectIsNeeded(t *testing.T) {
 		t.Run(test.command, func(t *testing.T) {
 			result, err := cli.RunCommand(ctx, args...)
 			assert.Error(t, err)
-			assert.Contains(t, result.Stderr, azdcontext.ErrNoProject.Error())
+			assert.Regexp(t, "no project exists; to create a new project, run `azd init`", result.Stderr)
```

Comment:
Should be a bad merge here `assert.Contains(t, result.Stderr, azdcontext.ErrNoProject.Error())` is what we want.
```suggestion
			assert.Contains(t, result.Stderr, azdcontext.ErrNoProject.Error())
```

Reply:
Fixed - bad merge.